spec/pods.md

```diff
@@ -159,7 +159,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
     * **app** (object, optional) substitute for the app object of the referred image's ImageManifest. See [Image Manifest Schema](aci.md#image-manifest-schema) for what the app object contains.
     * **mounts** (list of objects, optional) list of mounts mapping an app mountPoint to a volume. Each mount has the following set of key-value pairs:
       * **volume** (string, required) name of the volume that will fulfill this mount (restricted to the [AC Name](types.md#ac-name-type) formatting)
-      * **mountPoint** (string, required) name of the app mount point to place the volume on (restricted to the [AC Name](types.md#ac-name-type) formatting)
+      * **path** (string, required) path inside of the app filesystem to mount the volume; generally this will come from one of an apps mountPoint paths
```

Comment:
> generally this will come from one of an apps mountPoint paths

I think it's unclear what exactly this would look like to a user or how an implementation would handle it. For the sake of me understanding, is this supposed to be a no-op with the current e.g. rkt UX? That the mountPoint names might be exposed to users and would be used when an executor reifies a podmanifest?